app/src/main/java/it/niedermann/owncloud/notes/persistence/NotesRepository.java

```diff
@@ -608,18 +622,22 @@ public void modifyCategoryOrder(long accountId, @NonNull NavigationCategory sele
             final Context ctx = context.getApplicationContext();
             final SharedPreferences.Editor sp = PreferenceManager.getDefaultSharedPreferences(ctx).edit();
             int orderIndex = sortingMethod.getId();
-
+            final String categoryType = selectedCategory.getType().toString();
+            final String keyMetaCategory = categoryType.concat(String.valueOf(accountId));
             switch (selectedCategory.getType()) {
                 case FAVORITES: {
-                    sp.putInt(ctx.getString(R.string.action_sorting_method) + ' ' + ctx.getString(R.string.label_favorites), orderIndex);
+                    sp.putInt(ctx.getString(R.string.action_sorting_method) + ' ' + ctx.getString(R.string.label_favorites) + accountId, orderIndex);
+                    sp.putString(keyMetaCategory, "modified");
```

Comment:
What are the `sp.putString(...)` values needed for...?

Reply:
Ohh I see, I misunderstanded what you meant before. I would delete them.